src/index.html

```diff
@@ -137,18 +137,23 @@ <h1 id="video" class="video-title">Teaser</h1>
         <iframe id="promo-video" class="center-block" src="https://www.youtube.com/embed/voLw_IpG2l0" frameborder="0" allowfullscreen></iframe>
 
         <h1 id="sponsors" class="sponsors-title">Sponsors</h1>
-        <div class="row sponsor-images">
+        	<div class="row sponsor-images">
 					<div class="peta col-md-12">
-              <a href="https://www.imaginbank.com/" target="_blank"><img src="assets/img/imagin-bank-300.png"></a>
-          </div>
+              			<a href="https://www.imaginbank.com/" target="_blank"><img src="assets/img/imagin-bank-300.png"></a>
+          			</div>
 
-          <div class="giga col-md-12">
+          			<div class="giga col-md-12">
 						<a href="https://www.gft.com/" target="_blank"><img src="assets/img/GFT.png"></a>
 					</div>
 
+					<div class="giga col-md-12">
+						<a href="http://media.zurich.com/servizurich/index.htm" target="_blank"><img class="ml mr" src="assets/img/servizurich.png"></a>
+					</div>
```

Comment:
This is wrong!!!! Servizurich should be in the same div then GFT!!!!!!!!!

Reply:
They pay more than Skyscanner & Everis but less than GFT. I think that it's correct.

Reply:
yeah but they have the same img size either way. Only separation between GFT and ServiZurich are added. Which is too much spacing in my opinion